devops/setup-gpg/action.yaml

```
@@ -0,0 +1,48 @@
+name: 'Setup GPG'
+description: 'Configures this action to run gpg with a given key and pass'
+inputs:
+  private-key:  # id of input
+    description: 'gpg private key'
+    required: true
+  key-pass:  # id of input
+    description: 'gpg key pass'
+    required: true
+  key-name:  # id of input
```

```suggestion
  gpg-private-key:
    description: 'gpg private key'
    required: true
  gpg-key-pass:
    description: 'gpg key pass'
    required: true
  gpg-key-name:
```
Nit: make the variable names self documenting, and remove comments from code example in docs (assuming you were referring to this example: https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action#creating-an-action-metadata-file)

---

Not sure why we need interactive mode, because no one should know private key & passphrase.